exercises/toRoman/toRoman.js

```diff
@@ -10,19 +10,109 @@
  * toRoman(1);   // => 'I'
  * toRoman(6);   // => 'VI'
  * toRoman(152); // => 'CLII'
+ * 
+ * 
+ * 1 = I
+ * 5 = V
+ * 10 = X
+ * 50 = L
+ * 100 = C
+ * 500 = D
+ * 1000 = M
+ * 
+ * 
  *
  * @param {number} num - An integer
  * @returns {string} The input integer's Roman numeral representation
  */
 function toRoman(num) {
-  // This is your job. :)
+  let workingNum = num;
```

Comment:
Oop! I didn't see this. Looks like I was never tagged.

1. Consider using a structure like `[[1000, 'M'], [500, 'D'], ..., [1, I]]` to keep track of the value of each symbol.
1. Consider treating 4, 9, 40, 90, etc. as their own values whose symbols happen to consist of two characters.

By looking doing both of these, you can write this function in 4-5 lines using a single `for` loop.